### DISCUSSION TITLE
Github wont commit changes

### DISCUSSION THREAD

**Discussion by Light1yg** [community/community#65934] [New to GitHub]
**_I am new to GitHub and whenever it try to commit from the codespaces or from the repository_** but it just wont commits and loads infinitely I have Tried switching my browser and connecting From a different network but nothing does anyone why this might be happening

**Comment by Etzshally** [comment]
yes, you need to open bash in that directory check status, git logs. or you can just configure bash with username or email and then:

1) git commit -m "Your Commit"
2) git push (if you cloned the repo already and in it you are working)

**Comment by HamzaGorcevic** [comment]
Try writing git remote -v , to check if its on correct github repo. If not you can remove it, and then add new one

**Comment by frg** [comment]
Regarding issues with Github Codespaces, you can try the following.

1. Open the browser's developer console (typically F12 or Right-Click > Inspect on the page) and check for any JavaScript errors or warnings. This might give us a clue as to what's going wrong.
2. Browser extensions might be messing with the application. Try opening a browser window in incognito, and attempt to commit. If this works it might hint towards an issue with one of the browser extensions you have installed.
